Sources/SimultaneouslyScrollView/DefaultSimultaneouslyScrollViewHandler.swift

```diff
@@ -74,9 +101,26 @@ extension DefaultSimultaneouslyScrollViewHandler: UIScrollViewDelegate {
             return
         }
 
-        scrollViewsStore.allObjects
-            .filter { $0 != lastScrollingScrollView }
-            .forEach { $0.setContentOffset(scrollView.contentOffset, animated: false) }
+        for decorator in scrollViewsStore {
+            guard let registeredScrollView = decorator.scrollView else {
+                continue
+            }
+
+            if decorator.scrollView == lastScrollingScrollView {
+                continue
+            }
```

Comment:
Can we combine the `for .. in` with the preconditions inside the loop (`guard` & `if`) into a `.filter` and `.forEach` sequence? 🤔

Reply:
[ff3eba7](https://github.com/stonko1994/SimultaneouslyScrollView/pull/14/commits/ff3eba7325edd3e57c5f49b5910c063d5ac1c0c6)